### DISCUSSION TITLE
A15 VEIKK unable to detect tablet

### DISCUSSION THREAD

**Discussion by ghost** [OpenTabletDriver/OpenTabletDriver#1799] [Q&A] [answered]
I'm new to here so I don't exactly know if this is even the right place to put this, I'm just looking for help.

After being entirely unable to download the VEIKK drivers, I came to download this as an alternative. After downloading it easily, I found I can't detect my tablet, even after multiple attempts. I just wanna know if there's any troubleshooting to fix this issue. Please and thanks.

**Comment by jamesbt365** [comment, marked as answer]
your cable is probably faulty, you could check the device list in your os or help > export diagnostics in otd and send that tfile here as a .txt file.

I'm assuming this is solved with inactivity, but check the cable, port and make sure the version of otd is updated, you may need to remove veikk drivers with manual cleanup.
